tools/timelock_proposal/operations.go

```diff
@@ -0,0 +1,8 @@
+package timelock_proposal
```

Comment:
https://go.dev/blog/package-names

> They are lower case, with no under_scores or mixedCaps. 

We can just call it timelock

Reply:
gonna say timelock proposal because it doesn't have ALL of the timelock logic here (i.e. geth wrappers is in the getwrappers package)

Reply:
I was thinking our path could be more descriptive and look something like:

import(
   "github.com/smartcontractkit/ccip-owner-contracts/pkg/proposals/mcms"
   "github.com/smartcontractkit/ccip-owner-contracts/pkg/proposals/timelock"
)

That way the information about what the package does is described in the import path, while still sticking to golang conventions

Reply:
right but this wouldn't be accurate because both of the MCMS and Timelock geth wrappers are in an independent directory. We can restructure this in a future iteration, maybe

Reply:
The import path *is* the description of this library. The path suggested above describes that this is logic specific to timelock proposals, and mcms proposals.

Reply:
ohhhh I missed the `proposal` part, yeah this structure works for me